crates/viewer/re_data_ui/src/blob.rs

```diff
@@ -142,3 +157,46 @@ pub fn blob_preview_and_save_ui(
         });
     }
 }
+
+fn show_video_blob_info(
+    ui: &mut egui::Ui,
+    ui_layout: UiLayout,
+    video_result: &Result<re_renderer::video::Video, re_renderer::video::VideoError>,
+) {
+    match video_result {
+        Ok(video) => {
+            if ui_layout.is_single_line() {
+                return;
```

Comment:
I think we should show _something_ even in the single-line case. Ideally just everything, but compacted on one line (resolution, duration, codec).

Reply:
Maybe I'm confused, but I think if this says single line then this is the line of the blob which already shows the blob's size

<img width="287" alt="image" src="https://github.com/user-attachments/assets/4e412d6a-27d3-4374-b1d1-7931adcb9a22">

Reply:
Ah yes, of course.